pkg/postrender/exec.go

```diff
@@ -27,23 +27,24 @@ import (
 
 type execRender struct {
 	binaryPath string
+	args       []string
 }
 
 // NewExec returns a PostRenderer implementation that calls the provided binary.
 // It returns an error if the binary cannot be found. If the path does not
 // contain any separators, it will search in $PATH, otherwise it will resolve
 // any relative paths to a fully qualified path
-func NewExec(binaryPath string) (PostRenderer, error) {
+func NewExec(binaryPath string, args []string) (PostRenderer, error) {
```

Comment:
This is a public API so we cannot change the signature of the function.
You can either create a new function that will satisfy your requirement, or you will need to use the current one without changing its signature.

We need to do that to avoid breaking other programs using helm's API.

Reply:
i got. i had add a new function `NewExecWithArgs` and keep back the `func NewExec(binaryPath string) (PostRenderer, error) `
thanks.